src/plugins/Migrations/controllers/commands/migrateGuilds.js

```diff
@@ -0,0 +1,129 @@
+const db = require("quick.db");
+const dbh = require("../../../.common/structures/DataHandling/DatabaseHandler.js");
+const { Guild, Channel } = require("./../../../../../api/models");
+const migrationsHandler = require("./../../structures/MigrationHandler.js");
+
+exports.help = {
+  name: "migrateGuilds",
+  description: "Migrates all the data for the guilds",
+  usage: "migrateGuilds",
+  example: "migrateGuilds",
+};
+
+exports.conf = {
+  aliases: [],
+  cooldown: 5,
+  permissions: ["ADMINISTRATOR"],
+};
+
+exports.run = async (client, message, args) => {
+  const guilds = client.guilds.cache;
+  for (const guild of guilds) {
+    const guildId = message.guild.id;
+    const guildInDb = await Guild.findOne({ where: { id: guildId } });
+
+    if (guildInDb !== null) {
+      message.channel.send(`${guild[1].name} already has been created`);
```

Comment:
How many elements are in guild? What about element 0?

Reply:
guild[0] is the id, and guild[1] is the guildObject we get from discord. I have now made it more clear in the code, as well as fixed some inconsistencies.